ClientTCPWpfApp/MainWindow.xaml.cs

```diff
@@ -13,12 +13,9 @@ namespace ClientTCPWpfApp;
 /// </summary>
 public partial class MainWindow : Window
 {
-    private const string Ip = "178.159.53.223";
-    private const int Port = 8910;
-    private const int BufferSize = 256;
-    private IPEndPoint? _iPEndPoint;
-    private Socket? _listenSocket;
-    private bool _connected;
+    private const string _address = "127.0.0.1";
```

Comment:
Constants are always capitalized without underscores.
Rename to `Address` and `Port`.